src/components/folder/mainHeader.js

```diff
@@ -23,21 +31,37 @@ function MainHeader({ activeFolderId, onFolderClick }) {
   };
 
   return (
-    <FolderLinkHeader>
-      <MainNav>
-        <FolderList folderList={folderList} activeFolderId={activeFolderId} onFolderClick={handleFolderClick} />
-        <Link to="#">
-          <FolderAdd>
-            폴더 추가 <FolderAddIcon src={addIcon} alt="폴더 추가 아이콘" />
-          </FolderAdd>
-        </Link>
-      </MainNav>
-
-      <HeaderTitle>
-        <MainTitle>{activeFolderName || '전체'}</MainTitle>
-        <HeaderButtons>{activeFolderName && MAIN_ICON_BUTTONS.map(text => <MainIconButton key={text} text={text} />)}</HeaderButtons>
-      </HeaderTitle>
-    </FolderLinkHeader>
+    <>
+      <FolderLinkHeader>
+        <MainNav>
+          <FolderList folderList={folderList} activeFolderId={activeFolderId} onFolderClick={handleFolderClick} />
+          <Link to="#">
+            <FolderAdd>
+              폴더 추가 <FolderAddIcon src={addIcon} alt="폴더 추가 아이콘" />
+            </FolderAdd>
+          </Link>
+        </MainNav>
+
+        <HeaderTitle>
+          <MainTitle>{activeFolderName || '전체'}</MainTitle>
+          <HeaderButtons>
+            {activeFolderName &&
+              MAIN_ICON_BUTTONS.map(text => <MainIconButton key={text} text={text} toggleModal={() => toggleModal(text)} />)}
+          </HeaderButtons>
+        </HeaderTitle>
+      </FolderLinkHeader>
+
+      {modalType === '삭제' && (
+        <Modal toggleModal={() => toggleModal()} title="폴더 삭제">
+          <FolderDelete toggleModal={() => toggleModal()} folderName={activeFolderName}></FolderDelete>
+        </Modal>
+      )}
+      {modalType === '수정' && (
+        <Modal toggleModal={() => toggleModal()} title="폴더 이름 변경">
+          <FolderNameChange toggleModal={() => toggleModal()} folderName={activeFolderName} />
```

Comment:
```suggestion
        <Modal toggleModal={toggleModal} title="폴더 이름 변경">
          <FolderNameChange toggleModal={toggleModal} folderName={activeFolderName} />
```
[중요도: 낮음] JS의 경우는 이렇게 써도 된답니다 ㅎㅎ